README.md

```diff
@@ -0,0 +1,116 @@
+# Project TINA
+
+## Govhack 2024
+
+Project TINA is designed as an AI agent that sits between a constituent and any Local, State or Federal Government Department, Agency, or Service to provide a conversational interface that better facilitates the communication of constituent issues and the resolution of those issues.
+
+#### Authors
+
+[Daniel](dbryar@gmail.com) & [Louisa Bryar](louisa.bryar@gmail.com)
+
+## Introduction
+
+People _love_ to complain about government services. And not just the services, but the way their responsibilities are delivered within the context of the individual.
+
+> - _"Why can't the government fix all these potholes in our roads?"_
+> - _"How is it that the government can spend $5 billion on domestic violence prevention programs and I can still hear my neighbours at each other every night?"_
+> - _"That developer is cutting down all those beautiful trees. That can't be legal, can it?"_
+> - _"My kids want to join the local soccer team but I can't afford the club fees with the cost of food these days. Doesn't the government fund sports clubs?"_
+
+Most of the time though these complaints just boil down to: **This is not acceptable!**
+
+And you know what? Most of the time; it isn't.
+
+The problem on the other side (the government side), it that they _know_ all these complaints are mostly unfounded, or misdirected, and so hard to provide a non-specific (static) answer to - or even suggest a responsible party for - that it all just becomes overwhelming to online content managers. Even though websites are able to handle a large number of visitors and provide an enormous volume of information, given the depth and breadth of government services, the one thing the complainant needs becomes hidden amongst less relevant pages, much like the proverbial needle in a haystack.
```

Comment:
take out unfounded. unless there is evidence we should not say this. 
take out 'Online content managers"